assets/js/modules/analytics-4/components/settings/AdsConversionIDSettingsNotice.js

```diff
@@ -76,6 +101,7 @@ export default function AdsConversionIDSettingsNotice() {
 					a: (
 						<Link
 							href={ `${ settingsAdminURL }#/connected-services/ads` }
+							onClick={ () => trackConfirmNotificationEvent() }
```

Comment:
This doesn't need the anonymous function:

```suggestion
							onClick={ trackConfirmNotificationEvent }
```